packages/components/src/components/list-item/list-item.scss

```diff
@@ -172,6 +190,15 @@
 :host([scale="s"]) {
   .actions-start {
     padding-inline-end: var(--calcite-spacing-sm);
+    margin-inline-start: var(--calcite-internal-list-action-slot-spacing);
```

Comment:
There was an error in the design spec for List Item. This margin should be inline-end for both actions-start and actions-end. [Spec has been updated](https://www.figma.com/design/i6iep7fCM0yAHscPm6i7aG/Actions--Action-Bar--Action-Group-updates?node-id=3553-32901&t=TGwnt7fE5lKk5gJp-11).

Reply:
The notes in figma where wrong, but the examples are correct. 😆
<img width="1774" height="324" alt="CleanShot 2025-12-11 at 16 40 07@2x" src="https://github.com/user-attachments/assets/63119bb1-6166-48f6-beb5-d3e6dbefbdbb" />